### ISSUE TITLE
Grouped Window List: while dragging pinned icon, panel hid itself

### ISSUE THREAD

**Issue by LinuxOnTheDesktop** [linuxmint/alpha-testing#21]
**Issue**

&nbsp;&nbsp;&nbsp;&nbsp;I have a panel that is set to autohide. I use the Grouped Window List applet with that panel. I have various apps pinned via that applet. I dragged one of them. The panel hid itself - or temporarily disappeared for some other reason - during the drag (i.e. before I released the button). I managed to get a screenshot. Here it is (though it does not show the whole screen):

![image](https://user-images.githubusercontent.com/47056543/59479982-ca11b300-8e56-11e9-8b17-2fdf57d9710c.png)

The red dots one sees comprise the icon in question. (It's for the program called AngrySearch - which is excellent, and is perhaps a candidate for bundling with Mint. But I digress.)

&nbsp;&nbsp;&nbsp;&nbsp;Before I started using the Mint alpha, I did not see this problem. I have seen the problem only once.

**My system**

&nbsp;&nbsp;&nbsp;&nbsp;I am on what is the latest PPA build at the time of writing.

```
OS: Linux Mint 19.1 Tessa x86_64 
Host: 20KHCTO1WW ThinkPad X1 Carbon 6th 
Kernel: 5.1.9-050109-generic 
Uptime: 7 hours, 47 mins 
Packages: 3195 (dpkg), 5 (snap) 
Shell: bash 4.4.19 
Resolution: 2560x1440 @ 60.01Hz 
DE: Cinnamon 4.0.10 
WM: Mutter (Muffin) 
WM Theme: Mint-Y-Dark-trans (Mint-Y-Dark) 
Theme: Mint-Y-Dark [GTK2/3] 
Icons: Paper [GTK2/3] 
Terminal: xfce4-terminal 
Terminal Font: InconsolataGo 12 
CPU: Intel i7-8550U (8) @ 4.000GHz 
GPU: Intel UHD Graphics 620 
Memory: 3703MiB / 15928MiB 
```

**Log**

[xsession-errors.txt](https://github.com/linuxmint/alpha-testing/files/3288827/xsession-errors.txt)

**Comment by LinuxOnTheDesktop** [comment]
The problem is worse than I reported. For, sometimes, simply clicking an item - on an auto-hiding panel - initiates a drag of it, a drag that persists once the panel has hidden itself and after I have released the touchpad. I have had this sort of problem on the non-alpha but it seems worse on the alpha. This sort of problem will, if experienced by many, cause (in aggregate) a great deal of irritation.
